src/components/Web3ReactManager/index.tsx

```diff
@@ -1,37 +1,61 @@
-import { useWeb3React } from '@web3-react/core';
-import styled from 'styled-components';
-import { web3ContextNames } from 'provider/connectors';
-import { useEagerConnect, useInactiveListener } from 'provider/providerHooks';
-import { useContext } from '../../contexts';
-import { useInterval } from 'utils';
+import { useWeb3React } from "@web3-react/core";

```

Comment:
I think your formatter may be incorrect in this file, unless we changed the configs?